Add navbar(non-functional atm), delete other paragraphs and image to …

ethanimooney opened the pull request:
…make room to update. Currently adding the main index.html functionality, more commits to come

ethanimooney commented:
Sorry about the large-ish commit there, got carried away and forgot to commit earlier. 

Adds flexbox functionality to entire site, adds about me section with pictures, bio, and skills/projects section. Also adds connect buttons, links to 4 social media.

I am sure some stuff violates the style guides, I plan on fixing that in another PR, but feel free to point out violations as you see them.

shaquille-hall commented on portfolio/pom.xml:
```diff
@@ -8,6 +8,7 @@
   <version>1</version>
   <packaging>war</packaging>
 
+
```
Perhaps you can revert this file. There doesn't seem to be anything changing.

ethanimooney replied:
Got it. Added something to make re-running the project easier and it didn't work so I deleted it, must have left a few spaces or something.

shaquille-hall commented on portfolio/src/main/webapp/index.html:
```diff
@@ -3,18 +3,79 @@
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/75dadd3f2b.js" crossorigin="anonymous"></script>
```
HTML loads documents from top to bottom. Because of this, it's a common practice to add script tags just above your closing body tag. This avoids page render delays due to expensive js.

ethanimooney replied:
Ah okay. The walk through had it up there but I'll move it down.

shaquille-hall commented on portfolio/src/main/webapp/index.html:
```diff
@@ -3,18 +3,79 @@
+                     <img src="/images/ethan-p6.JPG" width="100px">
+                </div>
+                <div class="about-item-left-item">
+                    <p>
+                        Hey there! I'm Ethan Mooney, an aspiring software engineer from Edmond, Oklahoma.</br>
+                        I am studying Computer Science with Business at Oklahoma Christian University,
+                        and I am currently a STEP intern at Google.
+                        </br>
+                        </br>
+                        My main hobbies include photography, gaming, technology repair, and bouldering.</br>
+                        Channeling my inner nerd, I also co-lead a Pokemon TCG league at a local card shop.
+                        </br>
+                        </br>
+                        I began programming in 7th grade, when I was introduced to Scratch. I really enjoyed
+                        it, so once I got to my sophomore year of high school, I had the chance to enroll
+                        in a local vocational technology center's Computer Science Academy. This was where
+                        my love of coding really took hold and I learned enough to make me decide to pursue
+                        computer science as a degree and a career.
+                    </p>
```
The line length will expand for as much as the browser window allows. This will particularly be the case for the last paragraph. If you'd like, one solution may be to wrap the text using css.

ethanimooney replied:
I think it is wrapping most of it. I only have the few </br> at the end of some lines because I wanted them on a new line. the others are just for spacing.

shaquille-hall replied:
Ok. I see the corresponding css.

shaquille-hall commented on portfolio/src/main/webapp/index.html:
```diff
@@ -3,18 +3,79 @@
+                  </p>
+
+                  <h3>Projects</h3>
+                  <p>wheell [webapp that gives wheelchair users detailed information about searched locations]
+                      </br></br>
+                      udining.com [re-designed website for a local university dining company]
+                      </br></br>
+                      swablu [discord server management bot]
+                      </br></br>
+                      ethanmooney.me [personal website]
+                  </p>
```
Would a list be better here?

ethanimooney replied:
Yes it would -- changing

shaquille-hall replied:
Did you push the change? I'm not seeing it on my end.

ethanimooney commented:
Alright, I committed again. 

TL;DR: Addresses all PR comments, ensures the following of Google style guidelines, updates the first minor piece of work in the week 3 curriculum.

I went in and fixed the things you suggested, converted my CSS and HTML files to properly follow the Google styles guides (if only just to get in the habit of following them). Also tried to revert that pom.xml file back to original, removed an extra line.

Also started on the Week #3 curriculum so DataServlet.java was updated, but a very minor change..

shaquille-hall commented on portfolio/src/main/webapp/index.html:
```diff
@@ -25,12 +23,12 @@ <h1 class="header-item">Ethan Mooney</h1>
+                     <img src="/images/ethan-p3.jpeg" alt="Ethan by Google sign" width="100px">
+                     <img src="/images/ethan-p4.jpeg" alt="Ethan and CSSI friends" width="100px">
+                     <img src="/images/ethan-p5.jpeg" alt="Ethan in wheat field" width="100px">
+                     <img src="/images/ethan-p6.JPG" alt="Ethan CSSI graduation" width="100px">
```
nit: This file extension is inconsistent with the others.

ethanimooney replied:
Yeah idk why it did that, all came from the same place, can change.
